src/main/java/util/AwsUtil.java

```diff
@@ -47,23 +80,26 @@ public static boolean isCertificateCreated(String testNumber, String vin){
         AssumeRoleResult assumeResult =
                 stsClient.assumeRole(assumeRequest);
 
-        BasicSessionCredentials temporaryCredentials =
-                new BasicSessionCredentials(
-                        assumeResult.getCredentials().getAccessKeyId(),
-                        assumeResult.getCredentials().getSecretAccessKey(),
-                        assumeResult.getCredentials().getSessionToken());
-        String bucketName = loader.getS3Bucket();
+        return new BasicSessionCredentials(
+                assumeResult.getCredentials().getAccessKeyId(),
+                assumeResult.getCredentials().getSecretAccessKey(),
+                assumeResult.getCredentials().getSessionToken());
+    }
 
+    public static boolean certificateIsNotCreated(String testNumber, String vin){
+
```

Comment:
should this just be 
return !isCertificateCreated(testNumber, vin); ?

Reply:
yeah, so remember with us changing how long we wait for the certificate to be generated, that was making the pack run much longer because the tests that check for the certificate NOT to be present also used the same method and would take the max time, so i created a new method to use the old time we had for the NOT generated, but ive just realised i could have kept it the same and just passed in the iterations as a parameter, so i've changed it back and just passed the iterations in and the pack runs in less time now because its not waiting 5 mins each time to confirm a cert was not generated